test/js/helpers.unit.spec.js

```diff
@@ -504,30 +482,6 @@ describe('Schemaform helpers:', () => {
       expect(output.arrayField.length).to.equal(1);
       expect(output.emptyArray).to.be.undefined;
     });
-    it('should convert autosuggest field to id', () => {
```

Comment:
This functionality has been removed.

Reply:
I thought we still had autosuggest functionality?

Reply:
We do, but this was specifically testing whether autosuggest *data* is transformed and doesn't test the widget. As far as I can tell there are not any tests for the autosuggest widget functioning.